rtb/analisi-requisiti/src/descrizione.tex

```diff
@@ -1,3 +1,46 @@
 \section{Descrizione }
 \subsection{Obiettivi del prodotto}
+
```

Comment:
Mancano inoltre tutti i termini da inserire in glossario, per esempio prenotazione, ordinazione, recensione, ristoratore ecc..

Per aggiungere scrivere:

prenotazione$^{G}$

Reply:
controlli manualmente se non metti il riferimento di un termine al glossario più volte?